### PR TITLE
[Zookeeper] Add metric type mapping for server datastream

### PR DESCRIPTION
**Pull request by agithomas**
## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ]

## Related issues

https://github.com/elastic/beats/issues/8938
https://github.com/elastic/integrations/issues/7752

-

### CODE REVIEW

File: packages/zookeeper/data_stream/server/fields/fields.yml [hunk 1]
```diff
@@ -4,33 +4,41 @@
...
       description: Total number of nodes
+      metric_type: gauge
     - name: outstanding
       type: long
+      metric_type: gauge
       description: Number of requests queued at the server. This exceeds zero when the server receives more requests than it is able to process
     - name: received
       type: long
       description: Number of requests received by the server
+      metric_type: gauge
```
Comment (ritalwar) [inline packages/zookeeper/data_stream/server/fields/fields.yml:37]: Shouldn't this be counter?
same with sent count.

Comment (agithomas) [reply packages/zookeeper/data_stream/server/fields/fields.yml]: Verified and corrected

image

Comment (agithomas) [reply packages/zookeeper/data_stream/server/fields/fields.yml]: image

**Review by ritalwar — APPROVED** [review]
LGTM!

**Review by lalit-satapathy — APPROVED** [review]
LGTM

**Pull request merged**